stig/tui/tuiobjects.py

```diff
@@ -91,6 +92,7 @@ def _greedy_spacer():
 
 bottombar = Group(cls=urwid.Columns)
 bottombar.add(name='counters', widget=TorrentCountersWidget(), options='pack')
+bottombar.add(name='diskspace', widget=AvailableDiskSpaceWidget(), options='pack')
```

Comment:
Still need to figure out how to add a single-space separator between the two columns here, c.f. #146.

Reply:
Solved by adding `options='pack'`.